cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/runtimejob/DataprocJobMain.java

```diff
@@ -140,7 +140,7 @@ public static void main(String[] args) throws Exception {
       } finally {
         // call destroy() method on envProviderClass
         Method closeMethod = dataprocEnvClass.getMethod("destroy");
-        LOG.info("Invoking destroy() on {}", runtimeJobClassName);
+        LOG.info("Invoking destroy() on {}", dataprocEnvClassName);
```

Comment:
why is this change needed?

Reply:
This line was printed as `2021-12-10 03:06:51,302 INFO runtimejob.DataprocJobMain: Invoking destroy() on io.cdap.cdap.internal.app.runtime.distributed.runtimejob.DefaultRuntimeJob`
destroy() method does not exist in https://github.com/cdapio/cdap/blob/develop/cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/runtimejob/DefaultRuntimeJob.java but is actually in dataprocenvclass - https://github.com/cdapio/cdap/blob/develop/cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/runtimejob/DataprocRuntimeEnvironment.java#L102